Ninjathulhu/Ninjathulhu.Game/Entity.cs

```diff
@@ -68,6 +68,30 @@ public class EntityProperties
         public Dictionary<Type, ComponentProperties> FromPrefab = new Dictionary<Type, ComponentProperties>();
 
         public Dictionary<Type, ComponentProperties> FromSpawn = new Dictionary<Type, ComponentProperties>();
+
+        private static object Get(Dictionary<Type, ComponentProperties> properties, Type componentType, string propertyName)
```

Comment:
This is fair, I'm typically a fan of one class per file myself.  One thing I don't like here is that the Component-Entity core classes aren't off in their own directory, but eh, so far we're doing pretty good about not having too many files in any one place.